app/models/rentals.py

```diff
@@ -0,0 +1,20 @@
+from app import db
+from datetime import timedelta, datetime 
+
+
+class Rental(db.Model):
+    rental_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    customer_id = db.Column('customer_id', db.Integer, db.ForeignKey('customer.customer_id'))
+    video_id = db.Column('video_id', db.Integer, db.ForeignKey('video.video_id'))
+    check_out_date = db.Column('check_out_date', db.DateTime, default=datetime.now())
+    renter = db.relationship('Customer', backref='rentals', lazy=True, foreign_keys=customer_id)
+    video = db.relationship('Video', backref='rentals', lazy=True, foreign_keys=video_id)
+
```

Comment:
It can be helpful to add a check in date:

```suggestion
     check_in_date = db.Column('check_out_date', db.DateTime, default=datetime.now())
```